examples/simple-consumer.rb

```diff
@@ -0,0 +1,48 @@
+# Consumes lines from a Kafka partition and writes them to STDOUT.
+#
+# You need to define the environment variable KAFKA_BROKERS for this
+# to work, e.g.
+#
+#     export KAFKA_BROKERS=localhost:9092
+#
+
+$LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
+
+require "kafka"
```

Comment:
is there a reason? rubocop and other tools advice using single once. Also as I said - maybe it is worth considering adding rubocop to this gem. That way when more people contribute - code styling would be the same for all of contributors.